src/main/java/org/thoughtcrime/securesms/WebxdcStoreActivity.java

```diff
@@ -121,6 +121,15 @@ private WebResourceResponse interceptRequest(String url) {
     if (url.endsWith(".jpg") || url.endsWith(".jpeg") || url.endsWith(".png")) {
```

Comment:
@link2xt isn't better to just use the mimeType returned by core in HttpResponse?